bulk_site_creator/views.py

```diff
@@ -21,8 +26,50 @@
 @login_required
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Your code allows safe HTTP methods (i.e `GET`, `HEAD`, `OPTIONS`) and unsafe HTTP methods (i.e `POST`, `PUT`, `DELETE`) to perform an operation on a web application. This could impact its security.  We recommend that you to use safe http methods while performing read-only operations. If you have to use unsafe HTTP methods, use them only to change the state of an appilication. [Learn more](https://cwe.mitre.org/data/definitions/352.html).